### PR TITLE
[#7]Fix parallax jitter

### PR DESCRIPTION
**Pull request by jjannetty**
This could use a quick look. Not that I did much, but it did fix the jitter you saw @nmertaugh

### CODE REVIEW

File: app/assets/stylesheets/common/layout.scss [hunk 1]
```diff
@@ -7,3 +7,16 @@
 
 .l-ptb-s { padding-bottom: $small; padding-top: $small; }
 .l-ptb-m { padding-bottom: $medium; padding-top: $medium; }
+
```
Comment (nmertaugh) [inline app/assets/stylesheets/common/layout.scss]: Is this extra newline intentional?

File: app/assets/stylesheets/common/layout.scss [hunk 2]
```diff
@@ -7,3 +7,16 @@
 
 .l-ptb-s { padding-bottom: $small; padding-top: $small; }
 .l-ptb-m { padding-bottom: $medium; padding-top: $medium; }
```
Comment (nmertaugh) [inline app/assets/stylesheets/common/layout.scss]: I know this isn't part of your PR, but what is our position on single line style rules like this? If we're okay with them, when do we use them? Seems like we should apply them consistently or not at all. @jjannetty what do you think?

**Comment by nmertaugh** [comment]
Very minor comments @jjannetty them RTM :green_apple:

File: app/assets/stylesheets/common/layout.scss (hunk 2 above)
Comment (jjannetty) [inline app/assets/stylesheets/common/layout.scss]: yeah, i agree, I will make them multi-line

**Pull request closed without merging**